### DISCUSSION TITLE
Plotting not start I have a big problem.

### DISCUSSION THREAD

**Discussion by silahendaz** [madMAx43v3r/chia-plotter#291] [Q&A]
My processors is old. I have 2 processor on my motherboard  and I must choose "DİSABLE BİTFİELD PLOTTİNG" in chia.exe

-trying your plotter system but not working. 

Final Directory: G:\madmax2\
Number of Plots: 3
Process ID: 7008
Number of Threads: 8
Number of Buckets: 2^7 (128)
PS D:\PLOTCU>

not continue. powershell only waiting plotter not start. 

pls HELP

**Comment by SebMoore** [comment]
https://github.com/madMAx43v3r/chia-plotter/issues/88#issuecomment-858069892
It would appear you need bitfield unfortunately.
